### ISSUE TITLE
Can't install onlyoffice-desktopeditors_amd64.deb (4.8.7-109) on Ubuntu 18.04

### ISSUE THREAD

**Issue by q4a** [ONLYOFFICE/DesktopEditors#106]
Can't install onlyoffice-desktopeditors_amd64.deb (4.8.7-109) on Ubuntu 18.04.
Here is output from terminal (i hope russian letters from terminal is ok):
```
$ sudo dpkg -i onlyoffice-desktopeditors_amd64.deb 
[sudo] пароль для user: 
Выбор ранее не выбранного пакета onlyoffice-desktopeditors.
(Чтение базы данных … на данный момент установлено 315539 файлов и каталогов.)
Подготовка к распаковке onlyoffice-desktopeditors_amd64.deb …
Распаковывается onlyoffice-desktopeditors (4.8.7-109) …
dpkg: зависимости пакетов не позволяют настроить пакет onlyoffice-desktopeditors:
 onlyoffice-desktopeditors зависит от libcurl3, однако:
  Пакет libcurl3 не установлен.

dpkg: ошибка при обработке пакета onlyoffice-desktopeditors (--install):
 проблемы зависимостей — оставляем не настроенным
Обрабатываются триггеры для desktop-file-utils (0.23-1ubuntu3) …
Обрабатываются триггеры для mime-support (3.60ubuntu1) …
При обработке следующих пакетов произошли ошибки:
 onlyoffice-desktopeditors
```
Looks like outdated dependency on libcurl3, but Ubuntu 18.04 using libcurl4.

**Comment by ShockwaveNN** [comment]
@q4a yeah, thanks, it's a known issue, we working on resolution
Related https://github.com/ONLYOFFICE/DocumentServer/issues/271

**Comment by ShockwaveNN** [comment]
onlyoffice-desktopeditors  5.1.27-126 released with support of Ubuntu 18.04
